Dockerfile.local

```diff
@@ -0,0 +1,8 @@
+FROM golang:1.17-buster@sha256:3e663ba6af8281b04975b0a34a14d538cdd7d284213f83f05aaf596b80a8c725 as builder
+
+COPY . /src
+WORKDIR /src
+RUN CGO_ENABLED=0 make dist
```

Comment:
@mfridman Added `CGO_ENABLED=0` to be consistant with release build. 

Refer https://github.com/pressly/goose/blob/master/.goreleaser.yml#L9

Without that, it's not working with alpine based docker images. 

I have tested with this 
```
FROM alpine:3.15

COPY --from=builder /src/bin/goose-linux64 /usr/local/bin/goose
RUN chmod +x /usr/local/bin/goose
RUN apk --no-cache add curl
RUN curl -ksSL https://raw.githubusercontent.com/cloudflare/cfssl/master/certdb/sqlite/migrations/001_CreateCertificates.sql -o 001_CreateCertificates.sql
RUN goose sqlite3 ./data.db up
RUN goose sqlite3 ./data.db status
```
Giving following error if the image is alpine. The same binary is working perfect in debian based images. 
![2021-12-12_11-28](https://user-images.githubusercontent.com/834529/145702151-cbade2f7-f984-4ae7-88ea-91adf85e1b0b.png)

After adding `CGO_ENABLED=0`

![2021-12-12_11-26](https://user-images.githubusercontent.com/834529/145702179-9ab78a9d-99d8-483c-8e07-d02719cf0436.png)

Working in both debian and alpine based images.

Reply:
Thanks, iirc this might be related to https://github.com/golang/go/issues/28065 (I haven't kept up whether this is still the state of the world).